src/app/clients/clients.datasource.ts

```diff
@@ -1,18 +1,17 @@
 /** Angular Imports */
-import { CollectionViewer, DataSource } from '@angular/cdk/collections';
+import { CollectionViewer, DataSource } from "@angular/cdk/collections";
```

Comment:
can you revert this quote to double quote change?  Also I am seeing some space change that we can revert